src/base/system_parameter.c

```diff
@@ -4830,14 +4832,15 @@ static SYSPRM_PARAM prm_Def[] = {
    (char *) NULL,
    (DUP_PRM_FUNC) NULL,
    (DUP_PRM_FUNC) NULL},
-  {PRM_ID_OPTIMIZER_RESERVE_01,
-   PRM_NAME_OPTIMIZER_RESERVE_01,
-   (PRM_FOR_CLIENT | PRM_USER_CHANGE | PRM_HIDDEN),
-   PRM_BOOLEAN,
-   &prm_optimizer_reserve_01_flag,
-   (void *) &prm_optimizer_reserve_01_default,
-   (void *) &PRM_OPTIMIZER_RESERVE_01,
-   (void *) NULL, (void *) NULL,
+  {PRM_ID_MAX_HASH_LIST_SCAN_SIZE,
+   PRM_NAME_MAX_HASH_LIST_SCAN_SIZE,
+   (PRM_FOR_SERVER | PRM_TEST_CHANGE | PRM_SIZE_UNIT),
```

Comment:
PRM_TEST_CHANGE is for test_mode.  is it intended?

Reply:
I fixed it.